independent-projects/tools/devtools-common/src/main/java/io/quarkus/devtools/commands/handlers/CreateProjectCommandHandler.java

```diff
@@ -56,7 +56,7 @@ public QuarkusCommandOutcome execute(QuarkusCommandInvocation invocation) throws
                 invocation.setValue(CLASS_NAME, className);
             }
 
-            final List<AppArtifactCoords> extensionsToAdd = computeCoordsFromQuery(invocation, extensionsQuery);
+            final List<AppArtifactCoords> extensionsToAdd = computeCoordsFromQuery(invocation, null, extensionsQuery);
```

Comment:
@maxandersen why don't we have the registry for creation too?

Reply:
My assumption is that @gastaldi just did it for add and list first.

But yes, it should be used during creation too in its final form.